authentication/authentication.py

```diff
@@ -0,0 +1,22 @@
+from django.utils.timezone import now
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.exceptions import AuthenticationFailed
+from rest_framework.authtoken.models import Token
+
+from forum import settings
+
+
+class ExpiringTokenAuthentication(TokenAuthentication):
```

Comment:
DjoserTokenAuthentification name would be better.

Reply:
Renamed